modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -208,6 +288,9 @@ const getRawTarget = (selection: EditorSelection): Node | null => {
 
 const drop = (state: Singleton.Value<State>, editor: Editor) => (e: EditorEvent<MouseEvent>) => {
   state.on((state) => {
+    if (state.intervalId.isSet()) {
+      state.intervalId.clear();
+    }
```

Comment:
If you look at the way `clear` works:
https://github.com/tinymce/tinymce/blob/c7e1029ff3fabd36d1a31a6d45c19509aa811fed/modules/katamari/src/main/ts/ephox/katamari/api/Singleton.ts#L25-L30

You don't need to check if it's set before calling it. By design it can be called no matter what the state is.
```suggestion
    state.intervalId.clear();
```